packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -105,6 +245,34 @@ export const useBadgeStyles = (state: BadgeState): BadgeState => {
     state.appearance === 'ghost' && styles.rootGhost,
     state.appearance === 'outline' && styles.rootOutline,
     state.appearance === 'tint' && styles.rootTint,
+    state.appearance === 'filled' && state.color === 'danger' && styles.rootFilledDanger,
+    state.appearance === 'outline' && state.color === 'danger' && styles.rootOutlineDanger,
```

Comment:
you might safe some bundle size by storing the values of these comparisons once before you use them, since you can minify a variable easily enough but not a string